release-note-script/src/main/java/MergeReleaseNotes.java

```diff
@@ -0,0 +1,242 @@
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * MergeReleaseNotes creates the body of the merged release note for
+ * ScalarDB. This script takes release note bodies for ScalarDB, ScalarDB
+ * Cluster, ScalarDB GraphQL, and ScalarDB SQL as markdown files as its
+ * input. And then output the merged release note body to standard
+ * output as a markdown.
+ * 
+ * This script is assumed to be executed in a GitHub Actions workflow.
+ * 
+ * Note that it is needed that java 11 to execute this script since it is
+ * executed as a Single-File Source-code program.
+ */
+public class MergeReleaseNotes {
+
+    private final String DEBUG = System.getenv("DEBUG");
+    private final String SECTION_SUMMARY = "Summary";
+
+    private Map<Edition, Map<Category, Map<Repository, List<ReleaseNote>>>> editionMap = new HashMap<>();
+
+    public static void main(String ... args) throws Exception {
+        if (args.length == 0) {
+            System.err.printf(
+                "Usage: java --source 11 %s.java scalardb.md cluster.md graphql.md sql.md\n",
+                MergeReleaseNotes.class.getSimpleName());
+        }
+
+        MergeReleaseNotes mergeReleaseNotes = new MergeReleaseNotes();
+        mergeReleaseNotes.createMergedReleaseNote(args);
+    }
+
+    void createMergedReleaseNote(String... args) throws Exception {
+        for (String path : args) {
+
+            Edition edition = null;
+            Repository repository = null;
```

Comment:
We don't need to initialize these with null:
```suggestion
            Edition edition;
            Repository repository;
```

Reply:
Done. Thank you.